flow-report/src/sidebar/sidebar.tsx

```diff
@@ -25,21 +27,22 @@ export const SidebarSummary: FunctionComponent = () => {
       <div className="SidebarSummary__icon">
         <SummaryIcon/>
       </div>
-      <div className="SidebarSummary__label">Summary</div>
+      <div className="SidebarSummary__label">{strings.summary}</div>
     </a>
   );
 };
 
 const SidebarRuntimeSettings: FunctionComponent<{settings: LH.ConfigSettings}> = ({settings}) => {
+  const strings = useUIStrings();
+
   return (
     <div className="SidebarRuntimeSettings">
       <div className="SidebarRuntimeSettings__item">
         <div className="SidebarRuntimeSettings__item--icon">
           <EnvIcon/>
         </div>
         {
-          `${settings.formFactor === 'desktop' ? 'Desktop' : 'Mobile'} | ` +
-          `${settings.screenEmulation.height}x${settings.screenEmulation.width}px`
+          settings.formFactor === 'desktop' ? strings.desktop : strings.mobile
```

Comment:
Does this need to be translated if we replace with device name?

Reply:
I suppose not but do we really want to do that? what do we say for desktop?

Reply:
If we are going to be consistent with [paul's mocks](https://github.com/GoogleChrome/lighthouse/pull/13125) we should include the device name. 

Looks like "Emulated Desktop"
https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/lib/i18n/locales/en-US.json#L2322

Reply:
Ah, yes let's be consistent! "Emulated Desktop" isn't really a device name ;)